content/engine/install/centos.md

```diff
@@ -33,12 +33,11 @@ CentOS versions:
 - CentOS 9 (stream)
 
 The `centos-extras` repository must be enabled. This repository is enabled by
-default, but if you have disabled it, you need to
-[re-enable it](https://wiki.centos.org/AdditionalResources/Repositories).
```

Comment:
Note for reviewer: this link is a 404 and didn't include much helpful info about this topic. I opted to not explain how to do this, but wanted feedback on if we should include the commands for how to enable repositories using `yum` or `dnf`.

Reply:
Related issue: https://github.com/docker/docs/issues/18364

Reply:
I'm OK with removing the link!